### ISSUE TITLE
Suggestion: crop telemetry data

### ISSUE THREAD

**Issue by zartom** [ericberman/MyFlightbookWeb#834]
For cases when you forget to finish a recording

**Comment by ericberman** [comment]
Yeah, I love this feature.  Hard to do because I support a wide variety of formats and can't write all of them (and if things are not time stamped - e.g., a lot of KML isn't - then what is "at the end"?)  But it also could be as simple as leaving the data unmodified and applying a crop to its display.

I also note that Foreflight has this (though they only support their own data).

**Comment by zartom** [comment]
If you forget to finish the recording, you can end up with a big file if you only hide it

Why not put it under "raw data" in the flight details view? Go to a specific line and choose "begin" or "end" and then trim it

**Comment by ericberman** [comment]
Right.  So the hard part is rewriting the data to actually remove the extraneous stuff, but that may be overengineering.  It may be a lot simpler to preserve a "window" and apply the filter at display time, leaving the original data intact but just cropping the display.
